.github/workflows/publish-dev.yml

```
@@ -22,7 +22,7 @@ jobs:
 
       - name: Publish
         run: |
-          npm version --git-tag-version=false $(jq --raw-output '.version' package.json).$(git rev-parse --verify HEAD)
+          npm version --git-tag-version=false $(jq --raw-output '.version' package.json).$(git rev-parse --short=7 --verify HEAD).$(date +%s)
```

```suggestion
          npm version --git-tag-version=false $(jq --raw-output '.version' package.json).$(git rev-parse --short HEAD).$(date +%s)
```
This should be sufficient

---

I'd like to keep `--verify` and the explicit 7 doesn't hurt no one.

---

Crawl asked for SHA 7 and removing =7 gives SHA 8. Also why remove --verify?

---

Might be possible that newer git versions have increased the default, in this case that should be fine. Taken from `man git-rev-parse`, `--short` implicitly enables `--verify` so there's no need to add it too.